### ISSUE TITLE
[Feature Request]: Ability to have unique menubars for each display

### ISSUE THREAD

**Issue by Xytronix** [jordanbaird/Ice#242]
### Check Existing Issues

- [X] I have searched existing issues, and this issue is not a duplicate

### Description

When using two monitors, there could be an option to show different menubars on each display.
I don't know if this would be possible.

This could be handy in combination with menubar widgets.

**Comment by jordanbaird** [comment]
I don't believe this is possible, unfortunately. macOS decides the exact behavior of this, but it essentially mirrors the active menu bar on all other displays. Once we have profiles and triggers, we _could_ make a trigger to assign different profiles to different displays, so that when one display activates, it switches to the corresponding profile.
